tests/blackbox/testlib/infra.py

```diff
@@ -121,3 +125,42 @@ def __exit__(self, exception_type, exception_value, traceback):
             if exception_value is None:
                 raise rexc
             raise rexc from exception_value
+
+
+def test_permissions(dbus_method, permissions, *args):
+    """
+    Test running dbus_method with and without root permissions.
+    param dbus_method: StratisDbus wrapper method
```

Comment:
Missing some colons here. Try running the doc target and see what happens. Likely it'll succeed but the docs will look funky.